src/legacy/ui/public/vis/editors/config/editor_config_providers.ts

```diff
@@ -128,37 +128,26 @@ class EditorConfigProviderRegistry {
     current: TimeIntervalParam,
     merged: EditorParamConfig,
     paramName: string
-  ): { timeBase?: string; default?: string } {
+  ): { timeBase: string; default: string } {
     if (current.default !== current.timeBase) {
       throw new Error(`Tried to provide differing default and timeBase values for ${paramName}.`);
     }
 
-    if (this.isTimeBaseParam(current) && this.isTimeBaseParam(merged)) {
+    if (this.isTimeBaseParam(merged)) {
       // In case both had where interval values, just use the least common multiple between both intervals
-      try {
-        const timeBase = leastCommonInterval(current.timeBase, merged.timeBase);
-        return {
-          default: timeBase,
-          timeBase,
-        };
-      } catch (e) {
-        throw e;
-      }
-    }
-
-    if (this.isTimeBaseParam(current)) {
-      try {
```

Comment:
:information_source: I removed the redundant `try-catch` blocks, that were just rethrowing the caught error.